tests/python/relay/test_op_qnn_dense.py

```diff
@@ -237,6 +246,20 @@ def test_qnn_dense_with_bias():
         qnn_dense_driver(int32_output_with_bias_params)
 
 
+def test_qnn_dense_without_bias_with_3d_input():
```

Comment:
Yeah, so the reason the test fails is because input shape is [1, 2, 10] and kernel shape is [3, 10] and these cannot broadcast (to be honest im not sure the semantics behind dense). Double check if the semantics behind the intention are correct.

Personally, I think these tests are actually unrelated to the main change so I'm fine just cutting this out if you want.

Reply:
Okay, let's cut them out. Should I rebase and push?

Reply:
Yeah i would be fine with that.

Reply:
done